app/models/job_invocation_composer.rb

```diff
@@ -75,6 +75,29 @@ def targeting_params
       api_params.slice(:targeting_type, :bookmark_id, :search_query).merge(:user_id => User.current.id)
     end
 
+    def triggering_params
+      raise ::Foreman::Exception, _('Cannot specify both recurrence and scheduling') if api_params[:recurrence].any? && api_params[:scheduling].any?
+
+      if api_params[:recurrence].any?
+        {
+          :mode          => :recurring,
+          :cronline      => api_params[:recurrence][:cron_line],
```

Comment:
Could you please avoid aligning hash :rocket:s? When we later change one line we have to realign all lines which makes harder to find the real change in git diff.